### PR TITLE
Switch Org - Add header, shimmers, zero state and fix search to ignore active org

### PR DESCRIPTION
**Pull request by yksolanki9**
https://www.loom.com/share/927f7cec1b544e09aaa8c4e181378570

Shimmers:
https://www.loom.com/share/f81dc53056394fe9976652f856cc2f58

### CODE REVIEW

File: src/app/auth/switch-org/switch-org.page.html [hunk 1]
```diff
@@ -1,71 +1,119 @@
...
-        matInput
-        placeholder="Search Orgs"
-      />
-    </mat-form-field>
+  <ion-toolbar mode="md">
+    <ion-buttons slot="start">
+      <ion-button>
+        <ion-icon src="../../../assets/svg/arrow-back.svg"></ion-icon>
+      </ion-button>
+    </ion-buttons>
```
Comment (yksolanki9) [inline src/app/auth/switch-org/switch-org.page.html:7]: Back navigation function to be implemented

File: src/app/auth/switch-org/switch-org.page.html [hunk 2]
```diff
@@ -1,71 +1,119 @@
...
+              svgIcon="fy-search"
+            ></mat-icon>
+            <input
+              #searchOrgsInput
+              [(ngModel)]="searchInput"
+              matInput
+              placeholder="Search for an org"
+              (focus)="onSearchBarFocus()"
+              autocomplete="off"
+            />
```
Comment (yksolanki9) [inline src/app/auth/switch-org/switch-org.page.html:72]: Search bar clear and focus methods to be implemented

File: src/app/auth/switch-org/switch-org.page.html [hunk 3]
```diff
@@ -1,71 +1,119 @@
...
+          class="switch-org__content-container__content__icon"
+          src="../../../assets/svg/fy-search.svg"
+          (click)="openSearchBar()"
+        ></ion-icon>
       </div>
-      <ion-icon class="switch-org__content-container__content__icon" src="../../../assets/svg/fy-search.svg"></ion-icon>
     </div>
 
-    <ng-container *ngIf="!isLoading">
+    <div [hidden]="!isSearchBarShown">
```
Comment (yksolanki9) [inline src/app/auth/switch-org/switch-org.page.html:53]: Search bar animation to be implemented

**Comment by abhishek1234321** [comment]
> https://www.loom.com/share/927f7cec1b544e09aaa8c4e181378570
> 
> Shimmers: https://www.loom.com/share/f81dc53056394fe9976652f856cc2f58

Hey, in your shimmers video, the shimmers go off before the orgs list is shown, could we fix it up?

File: src/app/auth/switch-org/switch-org.page.ts [hunk 4]
```diff
@@ -88,6 +88,20 @@ export class SwitchOrgPage implements OnInit, AfterViewInit, AfterViewChecked {
         }
       });
     }
+    this.activeOrg$ = this.offlineService.getCurrentOrg();
```
Comment (abhishek1234321) [inline src/app/auth/switch-org/switch-org.page.ts:91]: could someone land on this page when in offline mode?

**Review by abhishek1234321 — APPROVED** [review]

File: src/app/auth/switch-org/switch-org.page.ts (hunk 4 above)
Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.ts:91]: No

Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.ts:91]: So should I not cache it?

Comment (abhishek1234321) [reply src/app/auth/switch-org/switch-org.page.ts:91]: if this page is not accessible in offline mode then I think even if we cache it won't matter?

Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.ts:91]: yes

**Pull request merged**